themes/godotengine/pages/governance.htm

```diff
@@ -0,0 +1,84 @@
+title = "Governance"
+url = "/governance"
+layout = "more"
+description = "Governance model"
+is_hidden = 0
+==
+{##}
+{% put title %} Governance {% endput %}
+
+<div class="container">
+
+  <h3>Legal status</h3>
+  <p>
+    Godot is legally a part of the
+    <a href="https://sfconservancy.org">Software Freedom Conservancy</a>. As a
+    not-for profit charity, the Software Freedom Conservancy receives donations
+    and, based on their mission statement, use them for the benefit of the
+    project.
```

Comment:
```suggestion
    Godot is legally a part of
    <a href="https://sfconservancy.org">Software Freedom Conservancy</a>. As a
    not-for profit charity, Software Freedom Conservancy receives donations
    and, based on their mission statement, uses them exclusively for the benefit of the
    free and open source project.
```

Reply:
Small typo: "not-for-profit" or "non-profit"